docs/products/ontrack/project-epics/documentation-epic.md

```diff
@@ -0,0 +1,59 @@
+# Documentation Epic
+
+## Background / Context
+
+The OnTrack project has been worked on for many years by external consultants who have not contributed efficiently and effectively. As a result of this, documentation containing the information about design decisions, requirements and user information is not accurate or up to date within our OnTrack product.
+
+## Business Value
+
+To ensure that all work is usable,and the decision-making process is visible for future stakeholders, the business would like to retrospectively analyse any gaps in the documentation of our products and produce documentation. This will ensure previous decisions, design, and requirements are easily accessible to all future users, developers, and other stakeholders so relevant context is provided for all work in the future.
+
+## In Scope
+
+Design Documentation
+User documentation (high level)
+User guides (step by step how to)
+Requirement documentation
```

Comment:
**issue:** Could we convert this to an unordered list? This is rendered as one line at the moment.